cdap-app-fabric/src/main/java/io/cdap/cdap/internal/bootstrap/executor/AppCreator.java

```diff
@@ -94,12 +94,14 @@ private boolean appExists(ApplicationId applicationId) throws Exception {
   static class Arguments extends AppRequest<JsonObject> implements Validatable {
     private String namespace;
     private String name;
+    private boolean overwrite;
```

Comment:
if this isn't specified in the bootstrap config file, it will default to false right?

Reply:
Correct boolean defaults to false